main.go

```diff
@@ -52,6 +55,10 @@ func main() {
 			if c.Bool("show-all-syscalls") {
 				cfg.EventsToTrace = append(cfg.EventsToTrace, tracee.EventsNameToID["raw_syscalls"])
 			}
+			if c.Bool("security-alerts") {
+				cfg.EventsToTrace = append(cfg.EventsToTrace, tracee.EventsNameToID["mmap_alert"])
+				cfg.EventsToTrace = append(cfg.EventsToTrace, tracee.EventsNameToID["mprotect_alert"])
+			}
```

Comment:
I think these should be a single alert type, about memory protection. the fact that we implement it using mmap as a starting point is not important for the user.

Reply:
The user chooses "security-alerts", and doesn't have to know about mmap-alert and mprotect-alert. So, it's more flxible, and the user can ignore these event names